agent/stats/reporter/reporter.go

```diff
@@ -41,16 +39,10 @@ const (
 	// Default websocket client disconnection timeout initiated by agent
 	defaultDisconnectionTimeout = 15 * time.Minute
 	defaultDisconnectionJitter  = 30 * time.Minute
-	backoffMin                  = 1 * time.Second
-	backoffMax                  = 1 * time.Minute
-	jitterMultiple              = 0.2
-	multiple                    = 2
 )
 
 type DockerTelemetrySession struct {
-	s                    tcshandler.TelemetrySession
-	ecsClient            api.ECSClient
-	containerInstanceArn string
+	s tcshandler.TelemetrySession
```

Comment:
nit: as we have removed other two fields from this type, we can just make it an anonymous field, and then call `session.Start` at line 102.

Reply:
Sorry, didn't understand the comment. What exactly do you mean by anonymous field? Do you mind giving an example?

Reply:
change `s tcshandler.TelemetrySession` to `tcshandler.TelemetrySession`

Reply:
Got it. Sure. I can do that.

Reply:
I actually tried that, but the function gets in an infinite loop. I guess it is because both the functions - DockerTelemetrySession and tcshandler.TelemetrySession are named as Start. When I do session.Start, it tries to start itself and traps in an infinite loop. I would say for readability and understandability for noobs like me, we can have a member name rather than leaving anonymous. Let me know if you think otherwise. :)